### DISCUSSION TITLE
How to hide the close tab button when only one tab is opened?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#850] [Q&A] [answered]
## My problem:
- Hello Mr. black7375, I want to ask if you know how can I set auto-hide the "x" close tab button when only one tab is opened in Firefox.
- Also, how to set it to not count the Firefox View tab (button) as the second opened tab?

#### This is not a problem on Vivaldi, because there is already hidden the close tab button:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/fd566a80-1423-4d8c-9b54-5fe6c45f1a66)

#### Nor in Opera browser:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/86f9c0d5-caf6-409f-978c-10009454073c)

**Comment by Asad-13** [comment]
I believe the first is possible via userChrome CSS. I remember seeing that effect in other CSS themes. I'll try and search it up, but will need a few days as I'm not that well versed in CSS.

Although for the firefox view getting counted as a second tab, I believe that is inherently a property of firefox. Might be wrong, idk

**Comment by Aleksandra0056** [comment, marked as answer]
I already found a desired feature in the CSS plug-in called "CustomCSSforFx" (which was formerly part of Classic Theme Restorer) a long time ago, so I'm finally closing this discussion. (I don't even know why I kept this open).

By the way thanks all for help!

Closing as outdated.
